src/framework/standard/configuration.rs

```diff
@@ -273,14 +273,14 @@ impl Configuration {
         self
     }
 
-    /// Whether or not to respond to commands initiated with a mention. Note
-    /// that this can be used in conjunction with [`prefix`].
+    /// Whether or not to respond to commands initiated with the mentioned id.
+    /// Note that this can be used in conjunction with [`prefix`].
```

Comment:
While we are at it, turn this into another `**Note**`-block just like the note about ignoring mentions by default.